provider/internal/device/asa/resource.go

```diff
@@ -103,6 +106,14 @@ func (r *AsaDeviceResource) Schema(ctx context.Context, req resource.SchemaReque
 				MarkdownDescription: "The host used to connect to the device.",
 				Computed:            true,
 			},
+			"tags": schema.ListAttribute{
+				MarkdownDescription: "The tag associated with the device.",
```

Comment:
I was just reading the CDO documentation: we call tags labels in the documentation. So let's call them labels here in the provider (let's continue to call them tags in the client, because that's what they are internally).

Could you also please change the markdown description to: "Set a list of labels to identify the device as part of a group. Refer to the [CDO documentation](https://docs.defenseorchestrator.com/t-applying-labels-to-devices-and-objects.html#!c-labels-and-filtering.html) for details on how labels are used in CDO."